extensions/applicationinsights-react-js/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "noImplicitAny": true,
     "module": "es6",
     "moduleResolution": "Node",
-    "target": "es3",
+    "target": "es5",
```

Comment:
react doesn't work in IE, so this is a safe change (required for eslint validation)